src/spot_skills/src/spot_skills/spot_ros_wrapper.py

```diff
@@ -68,46 +68,40 @@ def __init__(self) -> None:
         rospy.loginfo(f"[{self._arm_action_name}] Action server has started.")
 
     def shutdown(self) -> None:
-        """Shut-down the ROS wrapper to Spot by safely powering off Spot."""
-        self._manager.shutdown()  # TODO: Don't stow the arm if holding something!
+        """Shut-down Spot's ROS wrapper by safely powering off Spot."""
+        self._manager.shutdown()
 
-    def handle_stand(
-        self,
-        request_msg: std_srvs.srv.TriggerRequest,
-    ) -> std_srvs.srv.TriggerResponse:
+    def handle_stand(self, request_msg: TriggerRequest) -> TriggerResponse:
         """Handle a service request to have Spot stand up.
 
-        :param      request_msg     Message representing a request for Spot to stand
+        :param request_msg: Message representing a request for Spot to stand
 
         :returns    Response conveying whether Spot has successfully stood up
         """
         del request_msg
         self._manager.stand_up(20)
-        return std_srvs.srv.TriggerResponse(True, "Spot is now standing.")
 
-    def handle_unlock_arm(
-        self,
-        request_msg: std_srvs.srv.TriggerRequest,
-    ) -> std_srvs.srv.TriggerResponse:
+        return TriggerResponse(True, "Spot is now standing.")
+
+    def handle_unlock_arm(self, request_msg: TriggerRequest) -> TriggerResponse:
         """Handle a service request to enable ROS control of Spot's arm.
 
-        :param      request_msg     Message representing a request to unlock Spot's arm
+        :param request_msg: Message representing a request to unlock Spot's arm
 
         :returns    Response conveying that Spot's arm has been unlocked
         """
         del request_msg
         self._arm_locked = False
-        return std_srvs.srv.TriggerResponse(True, "Spot's arm is now unlocked.")
+        self._arm_controller.unlock_arm()
+
+        return TriggerResponse(True, "Spot's arm is now unlocked.")
 
-    def handle_stow_arm(
-        self,
-        request_msg: std_srvs.srv.TriggerRequest,
-    ) -> std_srvs.srv.TriggerResponse:
+    def handle_stow_arm(self, request_msg: TriggerRequest) -> TriggerResponse:
         """Handle a service request to stow Spot's arm.
 
         TODO: If Spot is believed to be holding something, prevent stowing.
```

Comment:
Address or delete.

Reply:
Unimportant, as we are never currently calling this function. Leaving TODO to mark the concern.